### PR TITLE
Generalize KPIs

### PR DESCRIPTION
**Pull request by simenheg**
Generalize KPIs to make their functionality reusable for result indicators and key figures.

This has several implications:

* KPIs are no longer limited to three per organization/department/ product.

* The restriction on KPI types (users, satisfaction, and conversion) is lifted.

* Undefined/missing KPIs are no longer displayed on the front page, since users are free to have fewer than (or more than) three KPIs.

* KPI value display formats can now be customized instead of being locked to a type. The old formats (integer display for users, percentage display for satisfaction and conversion) are still kept for backwards compatibility with existing KPIs.

* KPIs lose their icons, which were associated with the three pre defined types.

### CODE REVIEW

**Review by petterhj — APPROVED** [review]
LGTM!

File: src/components/ItemRow.vue [hunk 1]
```diff
@@ -27,14 +27,14 @@
...
+          v-for="kpi in kpis"
+          :key="kpi.id"
+          v-tooltip="kpi.name"
           class="item__kpi"
-          :class="{ disabled: getKpiValue(id) === '–––' }"
+          :class="{ disabled: _formatKPIValue(kpi) === '–––' }"
         >
-          <i class="item__kpi-icon far" :class="kpi.icon" />
-          <span class="item__kpi-value">{{ getKpiValue(id) }}</span>
+          <i class="item__kpi-icon far fa-chart-bar" />
```
[the remark this answers is not in the document]
Comment (simenheg) [reply src/components/ItemRow.vue:36]: Ja, enig.

File: src/components/IndicatorTooltip.vue [hunk 2]
```diff
@@ -38,8 +38,8 @@ export default {
       type: Number,
       required: true,
     },
-    type: {
-      type: String,
+    kpi: {
+      type: Object,
       required: false,
```
[the remark this answers is not in the document]
Comment (simenheg) [reply src/components/IndicatorTooltip.vue:43]: Jepp.

File: src/components/KpiLink.vue [hunk 3]
```diff
@@ -22,20 +18,12 @@ export default {
   },
 
   props: {
-    kpiType: {
-      type: String,
-      required: true,
-    },
     kpi: {
       type: Object,
       required: false,
```
[the remark this answers is not in the document]
Comment (simenheg) [reply src/components/KpiLink.vue:23]: Jo. ;)

**Pull request merged**